lib/base/Domain.d.ts

```diff
@@ -1,8 +1,9 @@
-import TwilioClient = require('../rest/Twilio');
-import { RequestOpts } from "../base/BaseTwilio";
+import {BaseTwilio, RequestOpts} from "../base/BaseTwilio";
```

Comment:
Could be `import {BaseTwilio, RequestOpts } from "./BaseTwilio"; `

Reply:
Fixed!